### PR TITLE
[wordpressmadara] add support for new chapter ajax endpoint (backward compatible)

### LINKED ISSUE #3785: [IsekaiScan] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Don´t load any chapters of this conector

**To Reproduce**
1. Name of connector
2. Name of manga (if connector update is working)
3. Name of chapter (if manga update is working)
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)

**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17]

### LINKED ISSUE #3836: [MoreNovel] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Can't download and see the prewiew of chapter

**To Reproduce**
1. MoreNovel
2. All list of novel

**Screenshots**
hk1
hk2

**HakuNeko (please complete the following information):**
 - Version [8.3.4]

### LINKED ISSUE #3854: [Manytoon.com] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
i can get the list of manhwa but when i  click on required manhwa it shows chapter list 0

**To Reproduce**
1. manytoon.com
2. this problem is with all manhwa
3. Name of chapter (if manga update is working)
4. C0
5. fetch("https://manytoon.com/wp-admin/admin-ajax.php", {"credentials":"omit","headers":{"content-type":"application/x-www-form-urlencoded","sec-ch-ua":"","sec-fetch-dest":"empty","sec-fetch-mode":"cors","x-referer":"https://manytoon.com"},"referrerPolicy":"no-referrer-when-downgrade","body":"action=manga_get_chapters&manga=182610","method":"POST","mode":"cors"});

**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17]
Screenshot (20)
Screenshot (21)

### LINKED ISSUE #3867: [Manga-Diyarı] thewebsitename
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
chapters list not listed

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17]

### LINKED ISSUE #3873: [LeviatanScans] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Same exact problem that ManhuaES had and IsekaiScan currently has. Update function works, all titles show, but absolutely no chapters as of today. 

**To Reproduce**
1. Name of connector - LeviatanScans
2. Name of manga (if connector update is working) - All
3. Name of chapter (if manga update is working) - All
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)

Screen Shot 2021-08-23 at 8 53 57 PM

**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17]

### PR DESCRIPTION
**Pull request by 27136773**

### CODE REVIEW

**Comment by 27136773** [comment]
**Tested with:**
- Immortal Updates (new endpoint)
- Anikiga (old endpoint)
- AkuManga (pure html)

File: src/web/mjs/connectors/templates/WordPressMadara.mjs [hunk 1]
```diff
@@ -56,23 +56,47 @@ export default class WordPressMadara extends Connector {
...
+        });
+        const data = await this.fetchDOM(request, this.queryChapters);
+        if(data.length) {
+            return data;
+        } else {
+            throw new Error('No chapters found (old ajax endpoint)!');
+        }
+    }
+
+    async _getChaptersAjaxNew(mangaID) {
```
Comment (Sheepux) [inline src/web/mjs/connectors/templates/WordPressMadara.mjs]: I think that you can omit the "new" in the function name, but let old in the "old one"

**Review by Sheepux — APPROVED** [review]
/LGTM, minor debate on function name

File: src/web/mjs/connectors/templates/WordPressMadara.mjs (hunk 1 above)
Comment (27136773) [reply src/web/mjs/connectors/templates/WordPressMadara.mjs]: Done

**Pull request merged**